lib/LMSManagers/LMSHelpdeskManager.php

```diff
@@ -772,8 +772,8 @@ public function TicketAdd($ticket, $files = NULL) {
 			isset($ticket['netdevid']) && !empty($ticket['netdevid']) ? $ticket['netdevid'] : null,
 			isset($ticket['verifierid']) && !empty($ticket['verifierid']) ? $ticket['verifierid'] : null,
 			isset($ticket['deadline']) && !empty($ticket['deadline']) ? $ticket['deadline'] : null,
-			isset($ticket['service']) && !empty($ticket['service']) ? $ticket['service'] : null,
-			isset($ticket['type']) && !empty($ticket['type']) ? $ticket['type'] : null,
+			isset($ticket['service']) && !empty($ticket['service']) ? $ticket['service'] : -1,
+			isset($ticket['type']) && !empty($ticket['type']) ? $ticket['type'] : 9,
```

Comment:
Dlaczego tu liczby zamiast stałych?

Reply:
Co z uaktualnieniem danych w bazie danych, żeby dla NULL wstawić co trzeba?

Reply:
Czy stare tickety również przemieniać z null na odpowiednią wartość?

Reply:
To miałem na myśli.

Reply:
Poszła poprawka.